package.json

```diff
@@ -43,24 +43,24 @@
     "postcss": "^8.4.12",
     "postcss-dir-pseudo-class": "^6.0.4",
     "postcss-logical": "^5.0.4",
-    "prettier": "^2.6.0",
-    "puppeteer": "^13.5.1",
+    "prettier": "^2.6.1",
+    "puppeteer": "^13.5.2",
     "rimraf": "^3.0.2",
     "streamqueue": "^1.1.2",
-    "stylelint": "^14.6.0",
+    "stylelint": "^14.6.1",
     "stylelint-config-prettier": "^9.0.3",
     "stylelint-prettier": "^2.0.0",
     "systemjs": "^0.21.6",
     "systemjs-plugin-babel": "^0.0.25",
     "terser": "^5.12.1",
     "through2": "^4.0.2",
-    "ttest": "^3.0.0",
-    "typescript": "^4.6.2",
+    "ttest": "^4.0.0",
```

Comment:
Given that this is a *major* version increase, have you checked that the `stats/statcmp.js` script still works correctly without any to its code?

Reply:
Yes, it does. I'm not really sure why upstream decided to bump the major version here given that the commits between 3.0.0 and 4.0.0 at https://github.com/AndreasMadsen/ttest/commits/master don't seem to contain any API changes...